roles/info_displayer/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
         {{ hostvars.localhost.displayable_info|default([])
             + add_info }}
     ansible_connection: '{{ hostvars.localhost.ansible_connection }}'
-    ansible_python_interpreter: '{{ hostvars.localhost.ansible_python_interpreter }}'
+    ansible_python_interpreter: '{{ hostvars.localhost.ansible_python_interpreter|default(omit) }}'
```

Comment:
Let's make sure to land the fix in its own PR before merging this, to retain references etc.